### DISCUSSION TITLE
0.6.0 release allowing FastAPI 0.89+?

### DISCUSSION THREAD

**Discussion by phillipuniverse** [arthurio/fastapi-filter#244] [General]
Hello, I am a user of fastapi-filter, really awesome stuff! Has saved me a lot of time in writing this functionality myself!

I would like to use the automatic type hinting for response models for FastAPI, addd in 0.89.0. Unfortunately fastapi-filter 0.5.1 requires FastAPI < 0.89.

Looks like this has already been relaxed, just wondering when the plan is for a new release?

Thanks for such a great tool!

**Comment by phillipuniverse** [comment]
Looks like fixed in the 0.5.2 release https://github.com/arthurio/fastapi-filter/discussions/285

thanks!

**Comment by arthurio** [comment]
Sorry I'm just seeing this, even if I subscribed to all the github notifications I didn't receive anything ¯\_(ツ)_/¯

**Reply by phillipuniverse** [reply]
All good, glad we got a new release out! Thanks!!

GitHub does have a separate subscription for discussions at the repository level, might be you’re not subscribed to that and instead just direct mentions, or maybe just issues and PRs.

**Reply by arthurio** [reply]
Yeah, looks like I was getting only "github" notifications for discussions instead of "email" as well... Should be fixed now 👀
